scripts/bump-status-go.sh

```diff
@@ -60,60 +64,73 @@ if [[ "${STATUS_GO_VERSION}" = PR-* ]]; then
     STATUS_GO_VERSION="refs/pull/${STATUS_GO_VERSION#"PR-"}/head"
 fi
 
-# ls-remote finds only tags, branches, and pull requests, but can't find commits
-STATUS_GO_MATCHING_REFS=$(git ls-remote ${REPO_URL} ${STATUS_GO_VERSION})
-
-# It's possible that there's both a branch and a tag matching the given version
-STATUS_GO_TAG_SHA1=$(echo "${STATUS_GO_MATCHING_REFS}" | grep 'refs/tags' | cut -f1)
-STATUS_GO_BRANCH_SHA1=$(echo "${STATUS_GO_MATCHING_REFS}" | grep 'refs/heads' | cut -f1)
-
-REQUIRES_MSG=https://github.com/status-im/status-go/
-
-# Prefer tag over branch if both are found
-if [[ -n "${STATUS_GO_TAG_SHA1}" ]]; then
-    STATUS_GO_COMMIT_SHA1="${STATUS_GO_TAG_SHA1}"
-    REQUIRES_MSG=${REQUIRES_MSG}/tree/${STATUS_GO_VERSION}
-elif [[ -n "${STATUS_GO_BRANCH_SHA1}" ]]; then
-    STATUS_GO_COMMIT_SHA1="${STATUS_GO_BRANCH_SHA1}"
-    REQUIRES_MSG=${REQUIRES_MSG}/tree/${STATUS_GO_VERSION}
-elif [[ "${#STATUS_GO_VERSION}" -gt 4 ]]; then
-    STATUS_GO_COMMIT_SHA1="${STATUS_GO_VERSION}"
-    REQUIRES_MSG=${REQUIRES_MSG}/commit/${STATUS_GO_VERSION}
+if [ "$1" = "--latest" ]; then
+    STATUS_GO_VERSION=""
+    BRANCH_NAME=bump/status-go/latest-develop/${TIMESTAMP}
 else
-    echo "ERROR: Input not a tag or branch, but too short to be a SHA1!" >&2
-    exit 1
-fi
-
-echo "SHA-1 for ${STATUS_GO_VERSION} is ${STATUS_GO_COMMIT_SHA1}.
-Owner is ${STATUS_GO_OWNER}"
+    # ls-remote finds only tags, branches, and pull requests, but can't find commits
+    STATUS_GO_MATCHING_REFS=$(git ls-remote ${REPO_URL} ${STATUS_GO_VERSION})
+
+    # It's possible that there's both a branch and a tag matching the given version
+    STATUS_GO_TAG_SHA1=$(echo "${STATUS_GO_MATCHING_REFS}" | grep 'refs/tags' | cut -f1)
+    STATUS_GO_BRANCH_SHA1=$(echo "${STATUS_GO_MATCHING_REFS}" | grep 'refs/heads' | cut -f1)
+
+    REQUIRES_MSG=https://github.com/status-im/status-go/
+
+    # Prefer tag over branch if both are found
+    if [[ -n "${STATUS_GO_TAG_SHA1}" ]]; then
+        STATUS_GO_COMMIT_SHA1="${STATUS_GO_TAG_SHA1}"
+        REQUIRES_MSG=${REQUIRES_MSG}/tree/${STATUS_GO_VERSION}
+    elif [[ -n "${STATUS_GO_BRANCH_SHA1}" ]]; then
+        STATUS_GO_COMMIT_SHA1="${STATUS_GO_BRANCH_SHA1}"
+        REQUIRES_MSG=${REQUIRES_MSG}/tree/${STATUS_GO_VERSION}
+    elif [[ "${#STATUS_GO_VERSION}" -gt 4 ]]; then
+        STATUS_GO_COMMIT_SHA1="${STATUS_GO_VERSION}"
+        REQUIRES_MSG=${REQUIRES_MSG}/commit/${STATUS_GO_VERSION}
+    else
+        echo "ERROR: Input not a tag or branch, but too short to be a SHA1!" >&2
+        exit 1
+    fi
+
+    echo "SHA-1 for ${STATUS_GO_VERSION} is ${STATUS_GO_COMMIT_SHA1}.
+    Owner is ${STATUS_GO_OWNER}"
+
+    BRANCH_NAME=bump/status-go/${STATUS_GO_VERSION}/${TIMESTAMP}
+    COMMIT_MESSAGE="${COMMIT_MESSAGE}
 
+### Requires
+- ${REQUIRES_MSG}
+"
+fi
 
-TIMESTAMP=$(date +%s)
-BRANCH_NAME=bump/status-go/${STATUS_GO_VERSION}/${TIMESTAMP}
 
-git checkout master
+git checkout -q master
```

Comment:
maybe it's time to move `master` to variable?

Reply:
hmm, why? :think]